pom.xml

```diff
@@ -108,13 +108,13 @@
       -->
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-compress</artifactId>
-      <version>1.17</version>
+      <version>1.18</version>
```

Comment:
In appengine-plugins-core 0.6.8 this is still 1.17. It changed to 1.18 in 0.7.0. Will this be a conflict?

Reply:
`appengine-plugins-core`'s MANIFEST.MF specifies:
```
Import-Package: ...
  org.apache.commons.compress.archivers.tar;version="[1.17,2)",
  org.apache.commons.compress.archivers.zip;version="[1.17,2)",
  org.apache.commons.compress.compressors.gzip;version="[1.17,2)",
  org.apache.commons.compress.utils;version="[1.17,2)",
```
So we're good.